onnx/defs/nn/defs.cc

```diff
@@ -127,8 +141,15 @@ computes the output.)DOC";
             schema.Attr("strides",
                         "stride along each axis.",
                         AttrType::INTS);
+            schema.Attr("auto_pad",
```

Comment:
Do we want to bikeshed the attribute name here? :-) @ezyang @dzhulgakov

Reply:
I don't care about the name, it just need to be different than pads.  So let me know which name you guys prefer.